app/src/main/java/com/android/solvit/shared/model/request/analyzer/OkHttp.kt

```diff
@@ -0,0 +1,105 @@
+package com.android.solvit.shared.model.request.analyzer
+
+import android.util.Log
+import com.android.solvit.BuildConfig
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+import okhttp3.MediaType.Companion.toMediaTypeOrNull
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import okhttp3.RequestBody.Companion.toRequestBody
+import org.json.JSONArray
+import org.json.JSONObject
+
+suspend fun analyzeImagesWithOkHttp(imageUrls: List<String>): Triple<String, String, String> {
+  val client = OkHttpClient()
+
+  val predefinedUrls =
+      listOf(
+          "https://www.houseopedia.com/wp-content/uploads/2023/01/How-to-Fix-Four-Common-Toilet-Problems-e1674502258761.jpeg")
+
+  val systemMessage =
+      """
+    You are an AI that analyzes images and provides suggestions for service seekers.
+    Your task is to:
+    1. Classify each image into one of the following categories: 
+       PLUMBER, ELECTRICIAN, TUTOR, EVENT_PLANNER, WRITER, CLEANER, CARPENTER, PHOTOGRAPHER, 
+       PERSONAL_TRAINER, HAIR_STYLIST, OTHER. 
+    2. Generate a **Title** for the issue, written in the voice of the service seeker.
+    3. Generate a **Description** for the issue, written as if the seeker is describing the problem to potential providers hence analyze the photos joined and descripe the issue in the photos. 
+       
+       
+    Please return the results in the following format:
```

Comment:
You should be able to achieve this in a more elegant and reliable way using structured outputs, however we can address this in the next sprint

Reply:
Nice catch! I will definitely add this since it makes the code more accurate and independent  I'll check this later  👌